addons/sale_timesheet/models/project.py

```diff
@@ -374,6 +373,12 @@ def _compute_remaining_hours_so(self):
         for task in self:
             task.remaining_hours_so = mapped_remaining_hours[task._origin.id]
 
+    @api.depends('project_id.analytic_account_id', 'sale_order_id.analytic_account_id')
+    def _compute_analytic_account_id(self):
+        for task in self:
+            if not task.analytic_account_id or task.analytic_account_id == task.project_id.analytic_account_id:  # the second term is a bit like heuristic
```

Comment:
Why is the second term a bit like heuristic ?

Reply:
We completely changed the specs for that concern, thanks for your help !